sycl/include/CL/sycl/detail/pi.h

```diff
@@ -448,7 +448,8 @@ typedef enum {
 
 typedef enum {
   // Device-specific value opaque in PI API.
-  PI_MEM_ADVICE_UNKNOWN,
+  PI_MEM_ADVISE_RESET = 0,
+  PI_MEM_ADVICE_UNKNOWN = 1,
```

Comment:
Is this enum visible to the user?
I reckon this change breaks ABI.

Reply:
We should not change values in the enum, just add new

Reply:
I agree this does break the ABI. As a user visible enum is changing.
However, the current values do not conform to the SYCL2020 spec. So I believe that the enum values need to change to conform with the description of mem_advise "A value of 0 reverts the advice for ptr to the default behavior". 
Let me know if there is a better way of handling this without changing the value, many thanks?